Week3/homework/4-step3.js

```diff
@@ -1,2 +1,73 @@
 'use strict';
 // paste your freeCodeCamp solutions in here
+
+//instruction 1 -> create object using constructor function
+
+const motorBike = function () {
+    this.engines = 1;
+    this.wheels = 2;
+    this.seats = 2;
+};
+
+const theMotorBike = new motorBike();
+console.log(theMotorBike);
+
+//instruction 2 -> Give myCar a nickname property with a string value.
+
+const car = function () {
+    this.wheels = 4;
+    this.engines = 1;
+    this.seats = 5;
+};
+
```

Comment:
For your constructor functions, i.e. motorBike() and car(), because they are designed to be called to create a new object, convention is that they should be capitalised so:
MotorBike
Car
Would be considered better names.  Otherwise the code is excellent.

Reply:
I see, thanks for the information :+1: ( I fixed that after your review)